pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample

```diff
@@ -90,10 +90,13 @@ export SHARE_PASSWORD='password'
 #     size and not get the warning.
 #   * MUSIC_SIZE should be large enough to hold your music library, obviously.
 #   * If no music size is specified, no music drive will be created.
+#   * LIGHTSHOW_SIZE should be large enough to hold your Lightshow library, obviously.
+#   * If no lightshow size is specified, no music drive will be created.
 #   * If no boombox size is specified, no boombox drive will be created.
-export CAM_SIZE=30G
-#export MUSIC_SIZE=4G
-#export BOOMBOX_SIZE=100M
+export CAM_SIZE=80%
+export MUSIC_SIZE=15%
+export LIGHTSHOW_SIZE=2%
```

Comment:
This line should be the only change in the sample config file, and it should use a fixed size instead of a percentage. The rest of the file should remain unchanged.